src/view/reviewManager.ts

```diff
@@ -640,15 +640,17 @@ export class ReviewManager implements vscode.DecorationProvider {
 					}
 				}
 
-				const uri = vscode.Uri.parse(change.fileName);
+				const filePath = nodePath.resolve(this._repository.rootUri.fsPath, change.fileName);
+				const uri = vscode.Uri.file(filePath);
 				let changedItem = new GitFileChangeNode(
 					this.prFileChangesProvider.view,
 					pr,
 					change.status,
 					change.fileName,
 					change.blobUrl,
-					toReviewUri(uri, null, null, change.status === GitChangeType.DELETE ? '' : pr.head.sha, false, { base: false }),
-					toReviewUri(uri, null, null, change.status === GitChangeType.ADD ? '' : pr.base.sha, false, { base: true }),
+					change.status === GitChangeType.DELETE ?
```

Comment:
I'm still seeing issues comment behavior when browsing the "Changes in Pull Request" tree. Comments are now only showing up on the right side where the scheme is "file", not sure if that's the intended behavior? Finding the matched file still fails since filePath.query is checked before parentFilePath.query, but if those are reordered, then comments get duplicated between the left and ride side